resin/auth.py

```diff
@@ -105,9 +105,13 @@ def is_logged_in(self):
 
         """
 
-        return self.base_request.request(
-            '/whoami', 'GET', endpoint=self.settings.get('api_endpoint')
-        )
+        try:
+            self.base_request.request(
+                '/whoami', 'GET', endpoint=self.settings.get('api_endpoint')
+            )
+            return True
+        except:
```

Comment:
What exceptions ``base_request`` throws on error? I'd rather you only caught these specific exceptions, not *every* exception, as it's now (a plain ``except:`` catches all exceptions, regardless).